src/main/kotlin/server/Yegg.kt

```diff
@@ -37,10 +41,30 @@ object Yegg {
     const val ID_CHARS = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
     fun newID() = NanoId.generateOptimized(8, ID_CHARS, 61, 16)
 
+    private val coroutineScope = CoroutineScope(
+        SupervisorJob() +
+                Dispatchers.Default.limitedParallelism(1) +
```

Comment:
Is there a reason you use `Dispatchers.Default` instead of `Dispatchers.Main`?  It seems like `Main` is designed for your use case (making things happen in a single thread).